api/api.py

```diff
@@ -10,6 +10,14 @@
 
 app = Flask(__name__)
 
+status_FILTER_KEY = "status_filter"
+network_FILTER_KEY = "network_filter"
+alert_FILTER_KEY = "alert_filter"
+mag_type_FILTER_KEY = "mag_type_filter"
+event_FILTER_KEY = "event_filter"
+min_magnitude_FILTER_KEY = "min_magnitude_filter"
+continent_FILTER_KEY = "continent_filter"
```

Comment:
Constants should be all caps according to PEP 8

Reply:
True, I forgot to make that change. Will do so now